R/rc_initialise.R

```diff
@@ -1,38 +1,363 @@
 #' Function for initializing the RothC model
 #'
```

Comment:
is this function applicable for multiple fields only? it seems only applicable per field, and then its good to mention.